cloudera/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,44 @@
+name: Cloudera
+files:
+- name: cloudera.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

Comment:
This should be after check-specific options

Reply:
Nice catch!